### PR TITLE
Manually convert bulleted lists to Markdown

### PR DESCRIPTION
**Pull request by mbauman**
Bullet lists work just fine in Markdown right now, but it looks like the auto-converter didn't convert some.  So I just manually converted them as I was browsing through the helpdb.

### CODE REVIEW

File: base/docs/helpdb.jl [hunk 1]
```diff
@@ -4478,42 +4472,40 @@ Obtain the contents of an `IOBuffer` as a string, without copying. Afterwards, t
...
 Create a pipeline from a data source to a destination. The source and destination can
-be commands, I/O streams, strings, or results of other ``pipeline`` calls. At least one
+be commands, I/O streams, strings, or results of other `pipeline` calls. At least one
 argument must be a command. Strings refer to filenames.
 When called with more than two arguments, they are chained together from left to right.
-For example ``pipeline(a,b,c)`` is equivalent to ``pipeline(pipeline(a,b),c)``. This provides a more
+For example `pipeline(a,b,c)` is equivalent to `pipeline(pipeline(a,b),c)`. This provides a more
 concise way to specify multi-stage pipelines.
 
 **Examples**:
```
Comment (yuyichao) [inline base/docs/helpdb.jl]: Is this rendered correctly with MarkDown?

Comment (mbauman) [inline base/docs/helpdb.jl]: Yup, and it converts seamlessly back to RST, too.  I think it's a GitHub flavor extension.

Comment (yuyichao) [inline base/docs/helpdb.jl]: Hmm running `make -C doc` and checked the html doc, on master it looks like 
nacsctl-vs-office1

While on this branch it looks like:
nacsctl-vs-office1

The new version doesn't look too bad (honestly I prefer the new version) but this seems to be sth that markdown doesn't fully support (and / or not taken care of by the conversion code) and can introduce inconsistency in the html doc.

Comment (mbauman) [inline base/docs/helpdb.jl]: I could use a header instead, but I think it's fine (I thought you were talking about the nested ``` `` `backticks `` ```).

Comment (jakebolewski) [inline base/docs/helpdb.jl]: This doesn't look that bad.

**Comment by tkelman** [comment]
backported in #13072

**Pull request merged**